apps/amethyst/amethyst.talon

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
I'm wondering about the best way to handle window managers.

Maybe we can make stuff like i3 and amethyst opt-in by default, kinda like I did for the mouse grid?
https://github.com/knausj85/knausj_talon/blob/master/mouse_grid/grid_always.talon#L1
https://github.com/knausj85/knausj_talon/blob/a2830d844f505c37200a8f5dfdf7415cb2e3250b/settings.talon#L25

Reply:
Ah yeah, in public talon, I gate the commands on is_amethyst_running which worked well. Not sure how to do that now

Reply:
amethyst context is now only active if an app called amethyst is running (not necessarily focused)

Reply:
until we have a better way of detecting window manager, and in cases where the app running approach doesn't work,  I think having opt-in the same way that you did for mouse grid makes sense. I'll do it this way for i3wm when I revisit it